internalserver/handler.go

```diff
@@ -0,0 +1,104 @@
+package internalserver
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/pprof"
+	"sort"
+
+	"github.com/metalmatze/signal/healthcheck"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type Handler struct {
```

Comment:
Can we have documentation comments? It's super useful to to encourage usage of a new API I think